beam-core/Database/Beam/Backend/SQL/SQL2003.hs

```diff
@@ -34,6 +35,10 @@ class IsSql92FromSyntax from =>
                     -> Maybe Text
                     -> from
 
+class IsSql92OrderingSyntax ord =>
+    IsSql2003OrderingSyntax ord where
```

Comment:
I believe this ordering syntax is part of the extension T611: Elementary OLAP Operations. Would you mind renaming the class `IsSql2003OrderingElementaryOLAPOperationsSyntax` and adding a comment indicating the extension supported: 

```haskell
-- | Optional SQL2003 "Elementary OLAP operations" T611 support
```